gamestonk_terminal/common/quantitative_analysis/qa_view.py

```diff
@@ -590,9 +590,15 @@ def display_line(
 
     if log_y:
         ax.semilogy(data.index, data.values)
-        ax.yaxis.set_major_locator(plt.MaxNLocator(10))
-        ax.yaxis.set_major_formatter(ticker.FormatStrFormatter("$%.2f"))
-        ax.yaxis.set_minor_formatter(ticker.FormatStrFormatter("$%.2f"))
+        ax.yaxis.set_major_formatter(matplotlib.ticker.ScalarFormatter())
+
+        ax.yaxis.set_major_locator(matplotlib.ticker.LogLocator(base=10))
+        ax.yaxis.set_minor_locator(
+            matplotlib.ticker.LogLocator(base=10, subs=[2.0, 5.0])
+        )
+        ax.yaxis.set_minor_formatter(matplotlib.ticker.ScalarFormatter())
+        ax.ticklabel_format(style="plain", axis="y")
+
```

Comment:
I had played around with the LogLocator params because I really wanted more majorticks on charts like btc=f

here's my code:

```python
if log_y:
        ax.semilogy(data.index, data.values)

        ax.yaxis.set_major_locator(
            matplotlib.ticker.LogLocator(base=100, subs=[1.0, 2.0, 4.0, 6.0, 8.0, 10.0])
        )
        ax.yaxis.set_minor_locator(
            matplotlib.ticker.LogLocator(base=100, subs=[0.5, 3.0, 5.0, 7.0, 9.0])
        )

        ax.yaxis.set_major_formatter(matplotlib.ticker.ScalarFormatter())
        ax.yaxis.set_minor_formatter(matplotlib.ticker.ScalarFormatter())
        ax.ticklabel_format(style="plain", axis="y")
```

here are some results
![image](https://user-images.githubusercontent.com/11668535/148543250-310b3f1d-ab60-4056-a47e-64621e2939d2.png)
![image](https://user-images.githubusercontent.com/11668535/148543296-6c28a6d5-ec85-4fc9-8d8d-322096c135eb.png)
![image](https://user-images.githubusercontent.com/11668535/148543334-b0714f35-5fda-42a1-bd32-aa2b9509ad13.png)
![image](https://user-images.githubusercontent.com/11668535/148543370-f02b4f2b-adda-4c56-8c71-bca45ca8ae9b.png)
![image](https://user-images.githubusercontent.com/11668535/148543389-52dc15b8-e558-4b75-acfe-2f7092e5f00e.png)

Reply:
<img width="784" alt="Screenshot 2022-01-07 at 13 02 21" src="https://user-images.githubusercontent.com/25267873/148547679-04602a80-63d8-429e-9f1d-d38f3108f532.png">

it doesn't generalize well for TSLA 😞

Reply:
@DidierRLopes maybe remove the minor ticks at all?